tempto-ldap/src/main/java/com/teradata/tempto/fulfillment/ldap/LdapObjectDefinition.java

```diff
@@ -92,6 +101,12 @@ public LdapObjectDefinitionBuilder setAttributes(Map<String, String> attributes)
             return this;
         }
 
+        public LdapObjectDefinitionBuilder addModificationAttributes(Map<String, List<String>> modificationAttributes)
```

Comment:
Actually better name would be `setModificationAttributes`. Add suggest that you are merging previous value with one passed as parameter.
Same applies to `addObjectClassess` below. You can change that also if that is not huge pain.